### ISSUE TITLE
CA - NAK issues from CalREDIE

### ISSUE THREAD

**Issue by rnz6** [CDCgov/prime-reportstream#5874]
**Describe the bug**

From: Carroll, Ellaison@CDPH <Ellaison.Carroll@cdph.ca.gov>
Date: Thursday, June 16, 2022 at 12:15 AM
To: Green, Brick (CDC/DDPHSS/OD/HITSSU) (CTR) <rut5@cdc.gov>
Cc: Kumar, Anshul (CDC/OD/OCS) <rdz8@cdc.gov>
Subject: NAK issues from CalREDIE

Hey Brick,

Here are some of the errors we are seeing –

Missing OBR-2.1, OBR-3.1, ORC-2.1, ORC-3.1 and SPM-2.2.1

MyCare Labs^CDPH000087^CLIA  (accession example listed by the analyst has a typo, but she says there are 38 of them on 6/14 and 43 on 6/11, so hoping you can find some of them)

StemExpress^CDPH000087^CLIA  (two examples on 6/14 for this sender)

Missing ORC-23
Sonoma Academy^CDPH000085^CLIA    `0fdd7098-edfe-4123-bd78-20e79dffd31d`
 
OBX-17.1 > 20 characters
Altus Lab Wellness C^CDPH000075   `f2e71744-0fbd-4867-b69b-6530f243abe7`

Thank you,
Ellaison

**Comment by brick-green** [comment]
PR to fix OBX-17.1 truncation
https://github.com/CDCgov/prime-reportstream/pull/6024

**Comment by sliu1000** [comment]
@rnz6 Jessica, Please reach out to the CA and confirm that they are all set?

**Comment by rnz6** [comment]
Followed up with CA for confirmation 7/19

**Comment by sliu1000** [comment]
@brick-green 

Ellaison Carroll from Calredie, tried looking up the provided reportids and wasn't able to find them.  Do you have more documentation on the resend such as the accessions and when they were resetn (date/time), etc.

**Comment by brick-green** [comment]
Sorry about that @sliu1000. Those IDs above were for the whole report. I got mixed up. There were 150 test results transferred on July 18th between at 1:17 PT and 1:30 PT.

I'm not able to get the message IDs for those results though since I manually created and transferred them. When we do that manually we have to purge the files immediately afterward. If they still can't find the files with the time frame above I may be able to retrieve a few of the files names.
